docs/source/installing-brave.rst

```diff
@@ -1,184 +0,0 @@
-Installing Brave
```

Comment:
We should also add a redirect on this page instead of removing it.

If you Google for "Install Linux Brave" the first link is to [this page](https://brave-browser.readthedocs.io/en/latest/installing-brave.html) and it currently shows a 404.

Reply:
Good find, added.

![image](https://user-images.githubusercontent.com/907672/95352078-c6126b80-08b9-11eb-90a5-a4a2ca0a9d3b.png)